FirebaseCore/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Firebase 9.6.0
```

Comment:
I added this central CHANGELOG entry and ones on a per-product basis. I realize that the dependency on Installations means I could technically add an entry for every product that depends on Installations. Should I do that or this is fine or just keep the Firebase-wide entry in FirebaseCore? @paulb777 and @morganchen12

Reply:
My preference would be to only have this one and not duplicate for the products, but I'll defer to @morganchen12 to decide.

Reply:
+1, let's not duplicate. It is technically a pan-Firebase change, so if it regresses in any product, it's considered a bug.

Reply:
Thanks y'all. SGTM